src/proxy/http/profiles/mod.rs

```diff
@@ -280,3 +280,11 @@ impl fmt::Debug for Labels {
         self.0.fmt(f)
     }
 }
+
+impl Default for Labels {
+    fn default() -> Self {
+        let mut labels = IndexMap::with_capacity(1);
+        labels.insert("route".to_string(), "".to_string());
```

Comment:
Without more context, this is super weird. Why is there a magic string in here? Why is this necessary at all?

Reply:
Specifically, why is it necessary to hydrate an empty label?

Reply:
The problem is that if a route has no labels, its FmtLabels implementation will format empty which will cause the total label string to include `,,` which breaks Prometheus.  Adding an empty label is hack to work around this.